### ISSUE TITLE
length of custom entities?

### ISSUE THREAD

**Issue by kai-morich** [emsesp/EMS-ESP32#1699]
ems-esp version 3.6.5

while defining custom entities on the search for missing entities, I wondered about the value length, as I could not read a 4 byte int.

When configuring some non-existent test entities
![grafik](https://github.com/emsesp/EMS-ESP32/assets/25850372/59fc34c8-a445-4ee6-960b-63107023e704)
I got periodic scans

    017+10:34:56.900 N 815: [emsesp] Me(0x0B) -R-> boiler(0x08), ?(0x0DDD), length: 0x02 (offset 4)
    017+10:34:56.913 N 816: [emsesp] boiler(0x08) -W-> Me(0x0B), ?(0x0DDD), data: <empty> (offset 4)
    017+10:34:58.363 N 822: [emsesp] Me(0x0B) -R-> boiler(0x08), ?(0x0DDD), length: 0x01 (offset 8)
    017+10:34:58.376 N 823: [emsesp] boiler(0x08) -W-> Me(0x0B), ?(0x0DDD), data: <empty> (offset 8)
    017+10:34:59.678 N 828: [emsesp] Me(0x0B) -R-> boiler(0x08), ?(0x0DDD), length: 0x03 (offset 12)
    017+10:34:59.688 N 829: [emsesp] boiler(0x08) -W-> Me(0x0B), ?(0x0DDD), data: <empty> (offset 12)

I did not expect length 1 for int and 3 for long. Is this intended?

**Comment by proddy** [comment]
yes, INT and UINT (for unsigned) are both 1 byte/length with different ranges. It's int8 and uint8 in C/Java
SHORT/USHORT are 2 bytes (int16/uint16)
BOOL is 1 byte (uint8)
LONG is 3 bytes on the EMS bus, but stored as 4 in the ESP32 memory as an unsigned long (uint32)
